modules/oe_theme_content_person/templates/oe-theme-content-person-job-list.html.twig

```diff
@@ -6,20 +6,9 @@
  * @see \Drupal\oe_theme_content_person\Plugin\ExtraField\Display\PersonJobListExtraField::viewElements()
  */
 #}
-{% if simple %}
-  <ul class="ecl-unordered-list">
-    {% for item in items %}
-      <li class="ecl-unordered-list__item">
-        {{ item.label}}
-      </li>
-    {% endfor %}
-  </ul>
-{% else %}
-  {% for item in items %}
-    <div class="ecl-u-type-m ecl-u-type-bold ecl-u-mb-xs">{{ item.label }}</div>
-    {% if item.body %}
-      <div class="ecl-u-mb-xs">{{ item.body }}</div>
-    {% endif %}
-  {% endfor %}
-{% endif %}
-
+{% for item in items %}
+  <div class="ecl-u-type-m ecl-u-type-bold ecl-u-mb-xs">{{ item.label }}</div>
```

Comment:
This margin bottom (`ecl-u-mb-xs`) is causing a misalignment when there is no label.
<img width="610" alt="misalignment" src="https://user-images.githubusercontent.com/22908988/220060365-296035e5-fe39-4e8e-9af8-9fee3352bee0.png">